document-generation/utils/sampleTestData.ts

```diff
@@ -1547,3 +1548,92 @@ export const sampleRequirementsChecklistRequest = {
     },
   },
 };
+
+export const sampleJustificationAndApproval = {
+  documentType: "JUSTIFICATION_AND_APPROVAL",
+  templatePayload: {
+    fundingRequestFiscalYear: "FY23",
+    periodOfPerformance: {
+      requestedPopStartDate: "2023-06-01",
+      basePeriod: {
+        periodType: "BASE",
+        periodUnitCount: 12.0,
+        optionOrder: 1.0,
+        periodUnit: "MONTH",
+      },
+      optionPeriods: [
+        {
+          periodType: "OPTION",
+          periodUnitCount: 6.0,
+          optionOrder: 2.0,
+          periodUnit: "MONTH",
+        },
+        {
+          periodType: "OPTION",
+          periodUnitCount: 6.0,
+          optionOrder: 3.0,
+          periodUnit: "MONTH",
+        },
+      ],
+      recurringRequirement: true,
+      popStartRequest: false,
+      timeFrame: "NO_LATER_THAN",
+    },
+    cor: {
+      formalName: "GEN Who Where",
+      phoneAndExtension: null,
+      title: null,
+    },
+    otherContractingShopFullAddress: null,
+    title: "Jeff J&A Test 1",
+    jwccContractNumber: "HC105023D0002",
+    estimatedValue: "$0.00",
```

Comment:
Getting double dollar signs in locally generated file (multiple places), may need to change this sample data or the docx template.

![image](https://github.com/dod-ccpo/atat-web-api/assets/77642966/d46ae68c-54a7-4cb2-8442-c4610d28342f)